relay-server/src/utils/sizes.rs

```diff
@@ -26,7 +26,9 @@ pub fn check_envelope_size_limits(config: &Config, envelope: &Envelope) -> bool
             | ItemType::Transaction
             | ItemType::Security
             | ItemType::RawSecurity
-            | ItemType::FormData => event_size += item.len(),
+            | ItemType::FormData
+            | ItemType::ProfilingSession
+            | ItemType::ProfilingTrace => event_size += item.len(),
```

Comment:
I think this will have to be revisited, I am not sure if we can have the same expectations around profiling data's size.